client/src/AttendanceView/AttendanceView.js

```diff
@@ -574,7 +592,7 @@ export function AttendanceView() {
               {t('inputAttendance')}
             </Button>
           </div>
-          <div className="flex">
+          <div className="flex" style={filtersContainerStyle}>
```

Comment:
Don't use inline styles. You should add classes to the css.

Reply:
Done